.github/workflows/build-test-publish-images.yml

```diff
@@ -31,6 +31,28 @@ permissions:
   statuses: none
 
 jobs:
+  pr-builder:
+    if: ${{ !cancelled() && inputs.build_type == 'pull-request' }}
+    needs:
+      - checks
+      - compute-matrix
+      - build
```

Comment:
Are we missing `build-multiarch-manifest` here?

Reply:
No, it (I think intentionally) does not run on PRs: https://github.com/rapidsai/docker/pull/708#discussion_r1718693278

Reply:
@AyodeAwe what was the reason for not running `build-multiarch-manifest` on PRs any more as of #702?

I made some assumptions here but maybe those assumptions are wrong.

Reply:
`build-multiarch-manifest` was never usually run in PRs:

https://github.com/rapidsai/docker/blob/5cea518e52c6e795557a2f23ffa8320ce38cc783/.github/workflows/build-test-publish-images.yml#L194

Reply:
Ah! Ok great, thanks for checking my understanding there.

Reply:
I just pushed a commit making `build-multiarch-manifest` run on PRs.

I think it just makes everything simpler...:

* it can now be required by the `pr-builder` job (to be sure it isn't accidentally skipped)
* issues with it can be detected on PRs, instead of only on branch builds
* it can be unconditionally included in the `needs:` block for `delete-temp-images`

Those jobs tend to take like 5-30 seconds each and don't use GPUs. I think that's a pretty small price to pay in exchange for the benefits I listed above.